.editorconfig

```diff
@@ -0,0 +1,31 @@
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
+indent_style=space
+indent_size=4
```

Comment:
Using `editorconfig` file is a great idea, but please chage `indent_size=4` to `indent_size=2` here